### PR TITLE
complete various todos in tributary

### PR DESCRIPTION
**Pull request by akildemir**

### CODE REVIEW

File: coordinator/tributary/src/tendermint/mod.rs [hunk 1]
```diff
@@ -275,9 +275,14 @@ pub struct TendermintNetwork<D: Db, T: TransactionTrait, P: P2p> {
 
 pub const BLOCK_PROCESSING_TIME: u32 = 999;
 pub const LATENCY_TIME: u32 = 1667;
-// TODO: Add test asserting this
 pub const TARGET_BLOCK_TIME: u32 = BLOCK_PROCESSING_TIME + (3 * LATENCY_TIME);
 
+#[test]
+fn assert_target_block_time_no_overflow() {
```
Comment (kayabaNerve) [inline coordinator/tributary/src/tendermint/mod.rs]: This isn't about checking it doesn't overflow. It's about checking that the actual target block time *from the `tendermint-machine` crate* is equal to this.

File: coordinator/tributary/src/tendermint/tx.rs [hunk 2]
```diff
@@ -83,18 +83,15 @@ fn decode_and_verify_signed_message<N: Network>(
 }
 
 // TODO: Move this into tendermint-machine
-// TODO: Strongly type Evidence, instead of having two messages and no idea what's supposedly
-// wrong with them. Doing so will massively simplify the auditability of this (as this
-// re-implements an entire foreign library's checks for malicious behavior).
+// This function takes a TendermintTx, which can't be imported to tendermint create.
```
Comment (kayabaNerve) [inline coordinator/tributary/src/tendermint/tx.rs]: 1) This comment is contradictory, assuming it's part of the PR.
2) The underlying evidence checking can be moved.

File: coordinator/tributary/src/tendermint/mod.rs [hunk 3]
```diff
@@ -343,6 +348,7 @@ impl<D: Db, T: TransactionTrait, P: P2p> Network for TendermintNetwork<D, T, P>
 
     // add tx to blockchain and broadcast to peers
     // TODO: Make a function out of this following block
+    // Why? Seems like a single caller.
```
Comment (kayabaNerve) [inline coordinator/tributary/src/tendermint/mod.rs]: If it's still a single caller and doesn't appear to have common value, then likely an old comment we can remove :+1:

File: coordinator/tributary/src/tendermint/tx.rs (hunk 2 above)
Comment (akildemir) [reply coordinator/tributary/src/tendermint/tx.rs]: oh yes the evidence checking can be moved I thought TODO meant the whole function as is. 👍

File: coordinator/tributary/src/tendermint/mod.rs (hunk 1 above)
Comment (akildemir) [reply coordinator/tributary/src/tendermint/mod.rs]: yea was not sure either:D 👍

**Pull request merged**